client/pages/login/components/App.jsx

```diff
@@ -0,0 +1,15 @@
+var React = require('react');
+var ReactRouter = require('react-router');
+
+
+var Component = React.createClass({
+    render: function () {
+
+        return (
+            <ReactRouter.RouteHandler />
```

Comment:
This file was removed because 'RouteHandle` is gone:
https://github.com/reactjs/react-router/blob/master/upgrade-guides/v1.0.0.md#routehandler